perun-core/src/main/java/cz/metacentrum/perun/core/api/TasksManager.java

```diff
@@ -244,5 +244,13 @@
 	 */
 	List<Task> listAllTasksInState(PerunSession perunSession, Task.TaskStatus state) throws PrivilegeException;
 
+	/**
+	 * Suspends tasks propagation to engine.
+	 *
+	 * @param perunSession
+	 * @param val True to suspend propagation, false to resume propagation
+	 * @throws PrivilegeException
+	 */
+	void suspendTasksPropagation(PerunSession perunSession, boolean val) throws PrivilegeException;
```

Comment:
Using something like `suspend` instead of `val` for variable name will be generally better. Please update it in a whole method chain.